RELEASE.md

```diff
@@ -1,5 +1,12 @@
 # Release notes
 
+## 2.0.2
```

Comment:
It appears the convention in this doc is to precede the version number with 'Release'. 
```suggestion
## Release 2.0.2
```